_data/navigation.yml

```diff
@@ -50,10 +50,18 @@ datascience:
         url: /datascience/dc_index/
       - title: "Pathways"
         url: /pathways/
-      - title: "FH-DaSL Courses"
+  - title: "Data Management"
+    children:
+      - title: "NIH Data Sharing"
+        url: /data_sharing/
+  - title: "Training"
```

Comment:
I think it's angry 'cos this is missing a `children:` whatsit:
```
 57   - title: "Training"
 58     children:
 59       - title: "DaSL Courses for the Hutch"
 60         url: /hutch_courses/
 61       - title: "All DaSL Courses"
 62         url: "https://hutchdatascience.org/training/"
```

Reply:
Just committed what I think will be a fix for this. At least the online yaml validator says so.

Reply:
Thanks all. ;)